src/main/java/com/cse/helium/app/models/Movie.java

```diff
@@ -47,4 +48,17 @@
   public Movie(){
     // default constructor
   }
-}
\ No newline at end of file
+
+  /**
+   * GetPartitionKey.
+   */
+  public static String getPartitionKey(String id) {
+    // validate id
+    if (!StringUtils.isEmpty(id) && id.length() > 5
+        && StringUtils.startsWithIgnoreCase(id, "tt")) {
+      int idInt = Integer.parseInt(id.substring(2));
```

Comment:
i know you're just moving code but is it possible parseInt will throw an exception?

Reply:
Ah good catch. I think the parameters (ultimately this id) are validated prior to the controller getting the call, through the ParameterValidator class. So this method should just be getting ints.

Reply:
ah makes sense